imap/xapian_wrap.cpp

```diff
@@ -392,7 +392,9 @@ static std::string detect_language(const struct buf *part)
 {
     std::string iso_lang;
     bool reliable = false;
-    CLD2::Language lang = CLD2::DetectLanguage(part->s, part->len, 1, &reliable);
+    CLD2::Language lang = CLD2::DetectLanguage(
+        part->s, part->len < INT_MAX ? static_cast<int>(part->len) : INT_MAX, 1,
+        &reliable);
```

Comment:
The `, 1,` at the end of that middle line is a little lost, visually.  I've been finding lately that if there isn't logical groupings to wrap function parameters into, or if those logical groupings are still too long, then one parameter per line ends up being the most readable wrapping for me.  And especially so when some of the parameters are complex expressions in their own right, like the ternary there.  What about something like this:

```suggestion
    CLD2::Language lang = CLD2::DetectLanguage(
        part->s,
        part->len < INT_MAX ? static_cast<int>(part->len) : INT_MAX,
        1,
        &reliable);
```

Reply:
ok, done.